actionmailer/CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+*   Allow override of base job class used for ActiveJob-powered mail delivery.
+
+    This allows you to catch errors in Job deserialization by using
+    `rescue_from` in your own base class.
+
+    `ActionMailer::Base.delivery_job_class = MyDeliveryJobClass`
+
```

Comment:
I put in a blank line like your other changelog entries with similar annotations. Hope that's okay!